Source/VisualRecognitionV4/VisualRecognition.swift

```diff
@@ -13,22 +13,41 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  **/
+
+/**
+ * IBM OpenAPI SDK Code Generator Version: 99-SNAPSHOT-be3b4618-20201201-153403
+ **/
+
 // swiftlint:disable file_length
 
 import Foundation
+#if canImport(FoundationNetworking)
+import FoundationNetworking
+#endif
 import IBMSwiftSDKCore
 
+public typealias WatsonError = RestError
+public typealias WatsonResponse = RestResponse
 /**
- Provide images to the IBM Watson&trade; Visual Recognition service for analysis. The service detects objects based on a
- set of images with training data.
+ IBM Watson&trade; Visual Recognition is discontinued. Existing instances are supported until 1 December 2021, but as of
+ 7 January 2021, you can't create instances. Any instance that is provisioned on 1 December 2021 will be deleted.
+ {: deprecated}
+ Provide images to the IBM Watson Visual Recognition service for analysis. The service detects objects based on a set of
+ images with training data.
  */
 public class VisualRecognition {
 
     /// The base URL to use when contacting the service.
     public var serviceURL: String? = "https://api.us-south.visual-recognition.watson.cloud.ibm.com"
 
+    /// Release date of the API version you want to use. Specify dates in YYYY-MM-DD format. The current version is
+    /// `2019-02-11`.
+    public var version: String
+
     /// Service identifiers
-    internal let serviceName = "WatsonVisionCombined"
+    public static let defaultServiceName = "watson_vision_combined"
```

Comment:
is this the correct default service name? also check assistant, nlu, compare comply. not sure what this should be for swift